.eslintrc.js

```diff
@@ -27,7 +27,8 @@ module.exports = {
         "plugin:@typescript-eslint/recommended"
       ],
       rules: {
-        "@typescript-eslint/no-var-requires": 0
+        "@typescript-eslint/no-var-requires": 0,
+        "@typescript-eslint/no-empty-function": 0
```

Comment:
Do we need to remove these rules globally or is it possible to just add an inline comment to ignore the rule where we violate it?

Reply:
I tried adding the inline rule for a few hours but could not get it to work as expected unfortunately. I agree with you that this is not ideal to have within the global rules. I will try again with the inline stuff and report back.